tool/gravity/cli/install.go

```diff
@@ -735,6 +763,7 @@ var InterruptSignals = signals.WithSignals(
 	os.Interrupt,
 	syscall.SIGTERM,
 	syscall.SIGQUIT,
+	syscall.SIGHUP,
```

Comment:
This should cover the missing case of SSH session closing and terminating the group.